### PR TITLE
Fix #57: Add test runner and some unit tests

### LINKED ISSUE #57: Add Unit Tests

### PR DESCRIPTION
**Pull request by jimmcgaw**

### CODE REVIEW

File: test/utils/tags.test.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+
+import { tagByPrefix, cleanedFields } from '../../src/utils/tags';
+
+describe('tags', function () {
+
+  describe('#tagByPrefix', function () {
```
Comment (elimisteve) [inline test/utils/tags.test.js]: @jimmcgaw So `#functionName` is the convention?

File: README.md [hunk 2]
```diff
@@ -68,3 +68,7 @@ We use [mocha](https://mochajs.org/) as the testing framework, with [chai](http:
 To run tests:
 
 ``` $ npm test ```
+
+and go tests:
+
+``` $ go build ```
```
Comment (elimisteve) [inline README.md]: @jimmcgaw `go test ./...`

Comment (elimisteve) [inline README.md]: Or `go test -v ./...`

File: test/utils/tags.test.js (hunk 1 above)
Comment (jimmcgaw) [inline test/utils/tags.test.js]: Yep, that's typically what I've seen used in the past and used myself.

File: README.md (hunk 2 above)
Comment (jimmcgaw) [inline README.md]: Haha, wow. https://pbs.twimg.com/profile_images/499655836067561475/m2IkGAUp_400x400.jpeg

Comment (elimisteve) [inline README.md]: lol :-)

**Comment by elimisteve** [comment]
@jimmcgaw :+1:

**Pull request merged**